upsertGitHubTag-f79c4f36-3e61-43f4-8f6c-0b2e0e9774d7/deployment/index.js

```diff
@@ -110,59 +82,42 @@ function processEvent(event, callback) {
     console.log('GitHub Payload');
     console.log(JSON.stringify(body));
     
-    const action = body.action;
-    
-    if (action != null) {
-        console.log('action is ' + action);
-        var path = process.env.API_URL;
-        if (action === 'added' || (action === 'created' && body.installation.repository_selection)) {
-            console.log('action is proceeding as new repository');
-            const username = body.sender.login;
-            const installationId = body.installation.id;
-            path += "workflows/path/service";
-            if (action === 'added' && 'repositories_added' in body) {
-                // App has been automatically added as a "future" repository when the app is installed to the organization as a whole
-                processRepoAddition(body.repositories_added, username, installationId, path, callback);
-            }
-            if (action === 'created' && 'repositories' in body) {
-                // App has been selected to be installed to repositories
-                processRepoAddition(body.repositories, username, installationId, path, callback);
-            }
-            
-        } else if (action === 'created') {
-            console.log('action is proceeding as new release');
-            if ('release' in body) {
-                // A release has been created for some repository
-                const repository = body.repository.full_name;
-                const username = body.sender.login;
-                const gitReference = body.release.tag_name;
-                const installationId = body.installation.id;
-                
-                var releasePostBody = {
-                       "gitReference": gitReference,
-                       "installationId": installationId,
-                       "repository": repository,
-                       "username": username
-                    };
-                path += "workflows/path/service/upsertVersion";
-                
-                callEndpoint(path, releasePostBody, (response) => {
-                    console.log(response);
-                    if (response.statusCode < 400) {
-                        console.info('Service ' + repository + ' updated successfully');
-                        callback(null);
-                    } else if (response.statusCode < 500) {
-                        // Client error, don't retry
-                        console.error(`Error updating workflow: ${response.statusCode} - ${response.statusMessage}`);
-                        callback(null);
-                    } else {
-                        // Server error, retry
-                        console.info('Retrying call');
-                        callback({ "statusCode": response.statusCode, "statusMessage": response.statusMessage });
-                    }
-                });   
+    var path = process.env.API_URL;
+
+    // A push has been made for some repository (ignore pushes that are deletes)
```

Comment:
What exactly is a delete event and why are we ignoring it? Does it mean if somebody deletes a branch, the branch will continue to exist on Dockstore?

Reply:
Yes exactly. We haven't decided how we would handle deletes yet so for now this is ignored.

Reply:
This seems like a departure from existing behavior, doesn't it?

Reply:
I thought that if a branch is deleted on GH it stays on Dockstore. Or maybe that is for tools only.

Reply:
I think it deletes them: https://github.com/dockstore/dockstore/blob/853fb797990ec557cf574f6d5e28d36acad5aff8/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java#L164

Reply:
Huh. I wonder if that works. I wonder if I wrote it.

Reply:
For a branch, it deletes them, at least thats what happened when I was checking out the hotfix 1.8.2 behavior. I didn't check what happens if a tag was deleted though...

Reply:
@denis-yuen should a delete branch request from GitHub also delete the branch on Dockstore?

Reply:
Yeah, I think if that's the current behavior we should match it.

Reply:
Ok I'll confirm that that is what we currently do and if it is will add this feature.

Reply:
Looks like that is current behaviour, will update.

Reply:
Sorry If it looked like I ignored people I swear GitHub didn't load any new messages from the link Charles sent onwards.